test/test_rsmaz.rb

```diff
@@ -12,11 +12,11 @@
   #end
   
   it "should compress 'the' to one byte" do
-    RSmaz.compress("the").length.must_equal 1
+    _(RSmaz.compress("the").length).must_equal 1
```

Comment:
Needed to remove the following deprecation warning such as:
```
DEPRECATED: global use of must_equal from test/test_rsmaz.rb:31. Use _(obj).must_equal instead. This will fail in Minitest 6.
```